util/env.go

```diff
@@ -0,0 +1,51 @@
+package util
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"golang.org/x/term"
+)
+
+func Getenv(key, fallback string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return fallback
+}
+
+func GetenvBool(key string) bool {
+	return strings.ToLower(Getenv(key, "false")) == "true"
+}
+
+func GetenvPrompt(key string, prompt ...string) string {
+	value := Getenv(key, "")
+	if value == "" {
+		if len(prompt) > 0 {
+			fmt.Printf("%s is not set. %s: ", key, prompt[0])
+		} else {
+			fmt.Printf("%s is not set. Please enter a value: ", key)
+		}
+		fmt.Scanln(&value)
+	}
+	return value
+}
+
+func GetenvPromptSecret(key string, prompt ...string) string {
+	value := Getenv(key, "")
+	if value == "" {
```

Comment:
Same, should we use a `while value == ""` ? To prevent mistakes.

Reply:
```suggestion
	while value == "" {
```